gnovm/cmd/gno/test.go

```diff
@@ -500,10 +512,25 @@ func runTestFiles(
 	n := gno.MustParseFile("main_test.gno", testmain)
 	m.RunFiles(n)
 
+	printedEvents := 0
+
 	for _, test := range testFuncs.Tests {
 		testFuncStr := fmt.Sprintf("%q", test.Name)
 
-		eval := m.Eval(gno.Call("runtest", testFuncStr))
+		res := gno.Call("runtest", testFuncStr)
+
+		eval := m.Eval(res) // NOTE: verbose prints get here
+		if printEvents {
```

Comment:
I agree with @moul, I think we should print these events if the `verbose` flag is enabled, and not hide this functionality behind an additional flag we need to now maintain

The reasoning is that events are now an integral part of the Gno development lifecycle, and not a "feature" we can disable

Reply:
> The reasoning is that events are now an integral part of the Gno development lifecycle

I totally understand, however for easier(or at least more readable) debugging flags needs to be separated.

Reply:
446eee7d57bd852f13faac818a7b56dd1436e1ba